### PR TITLE
Migrate to GitHub Actions for CI

### PR DESCRIPTION
**Pull request by parndt**
I decided to move our continuous integration to GitHub Actions.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+          POSTGRES_DB: postgres
+        ports:
+        - 5432:5432
+        # needed because the postgres container does not provide a healthcheck
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+env:
+  BUNDLE_JOBS: 4
+  POSTGRES_HOST: localhost
+  POSTGRES_PORT: ${{ job.services.postgres.ports[5432] }}
```
Comment (parndt) [inline .github/workflows/main.yml]:
```suggestion
  POSTGRES_PORT: 5432
```

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    steps:
+    - uses: actions/checkout@v1
+    - name: Setup Ruby
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby }}
+        architecture: 'x64'
+    - run: gem install bundler --version '>= 2.0.2'
+    - run: bundle install --without development
+    - run: bundle exec rspec --format documentation
```
Comment (parndt) [inline .github/workflows/main.yml]:
```suggestion
	- run: bin/rake refinery:testing:dummy_app
    - run: bin/rspec --format documentation
```

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    steps:
+    - uses: actions/checkout@v1
+    - name: Setup Ruby
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby }}
+        architecture: 'x64'
+    - run: gem install bundler --version '>= 2.0.2'
+    - run: bundle install --without development
+	- run: bin/rake refinery:testing:dummy_app
```
Comment (parndt) [inline .github/workflows/main.yml]:
```suggestion
    - run: bin/rake refinery:testing:dummy_app
```

File: .github/workflows/main.yml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        ruby: [ '2.6.x' ]
+    
+    steps:
+    - uses: actions/checkout@v1
+    - name: Setup Ruby
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby }}
+        architecture: 'x64'
+    - run: gem install bundler --version '>= 2.0.2'
```
Comment (parndt) [inline .github/workflows/main.yml]:
```suggestion
    - run: sudo apt-get install libpq-dev -y
    - run: gem install bundler --version '>= 2.0.2'
```

File: .github/workflows/main.yml [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+        ruby: [ '2.6.x' ]
+    
+    steps:
+    - uses: actions/checkout@v1
+    - name: Setup Ruby
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby }}
+        architecture: 'x64'
+    - run: sudo apt-get install libpq-dev -y
```
Comment (parndt) [inline .github/workflows/main.yml]:
```suggestion
    - name: install database support libraries
    - run: sudo apt-get install libpq-dev libmysqlclient-dev libsqlite3-dev -y
```

File: .github/workflows/main.yml [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+    - name: Setup Ruby
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby }}
+        architecture: 'x64'
+    - run: sudo apt-get install libpq-dev libmysqlclient-dev libsqlite3-dev -y
+    - run: gem install bundler --version '>= 2.0.2'
+    - run: bundle install --without development
+    - run: bin/rake refinery:testing:dummy_app
+    - run: bin/rspec --format documentation
```
Comment (parndt) [inline .github/workflows/main.yml]:
```suggestion
    - run: bin/rspec ./*/spec --format documentation
```

File: .github/workflows/main.yml [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+        ruby: [ '2.6.x' ]
+    
+    steps:
+    - uses: actions/checkout@v1
+    - name: Setup Ruby
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby }}
+        architecture: 'x64'
+    - run: sudo apt-get install libpq-dev libmysqlclient-dev libsqlite3-dev -y
```
Comment (parndt) [inline .github/workflows/main.yml]:
```suggestion
    - run: sudo apt-get install chromedriver libpq-dev libmysqlclient-dev libsqlite3-dev -y
```

File: .github/workflows/main.yml [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+        ruby: [ '2.6.x' ]
+    
+    steps:
+    - uses: actions/checkout@v1
+    - name: Setup Ruby
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby }}
+        architecture: 'x64'
+    - run: sudo apt-get install chromedriver libpq-dev libmysqlclient-dev libsqlite3-dev -y
```
Comment (parndt) [inline .github/workflows/main.yml]:
```suggestion
    - run: sudo apt-get install chromium libpq-dev libmysqlclient-dev libsqlite3-dev -y
```

File: .github/workflows/main.yml [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+        ruby: [ '2.6.x' ]
+    
+    steps:
+    - uses: actions/checkout@v1
+    - name: Setup Ruby
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby }}
+        architecture: 'x64'
+    - run: sudo apt-get install chromium libpq-dev libmysqlclient-dev libsqlite3-dev -y
```
Comment (parndt) [inline .github/workflows/main.yml]:
```suggestion
    - run: sudo apt-get install chromium-chromedriver libpq-dev libmysqlclient-dev libsqlite3-dev -y
```

File: .github/workflows/main.yml [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+        ruby: [ '2.6.x' ]
+    
+    steps:
+    - uses: actions/checkout@v1
+    - name: Setup Ruby
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby }}
+        architecture: 'x64'
+    - run: sudo apt-get install chromium-chromedriver libpq-dev libmysqlclient-dev libsqlite3-dev -y
```
Comment (parndt) [inline .github/workflows/main.yml]:
```suggestion
    - run: sudo apt-get update
    - run: sudo apt-get install chromium-chromedriver libpq-dev libmysqlclient-dev libsqlite3-dev -y
```

File: .github/workflows/main.yml [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+    
+    steps:
+    - uses: actions/checkout@v1
+    - name: Setup Ruby
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby }}
+        architecture: 'x64'
+    - run: sudo apt-get update
+    - run: sudo apt-get install chromium-chromedriver libpq-dev libmysqlclient-dev libsqlite3-dev -y
```
Comment (parndt) [inline .github/workflows/main.yml]:
```suggestion
    - run: sudo apt-get install chromium-chromedriver imagemagick libpq-dev libmysqlclient-dev libsqlite3-dev -y
```

**Comment by parndt** [comment]
Just need to support more ruby versions now

**Comment by parndt** [comment]
@bricesanchez it works! And it didn't even take me a year! 😂

**Review by simi — APPROVED** [review]

**Comment by parndt** [comment]
Thanks @bricesanchez and @simi !

**Pull request merged**